libr/debug/debug.c

```diff
@@ -343,6 +343,11 @@ R_API int r_debug_select(RDebug *dbg, int pid, int tid) {
 	if (dbg->h && dbg->h->select && !dbg->h->select (pid, tid))
 		return false;
 
+	{
+		char buf[32];
+		sprintf (buf, "pid %d", pid);
+		r_io_system (dbg->iob.io, buf);
```

Comment:
having a textual interface with the io plugins is useful, but ideally we should generalize the most common operations on io plugins and add more callbacks if needed. Those callbacks are wrapped by core r_io apis like`r_io_system` which checks if the current plugin handles that callback or not.